agent/src/beerocks/monitor/monitor_stats.cpp

```diff
@@ -432,7 +432,8 @@ void monitor_stats::process()
     }
 }
 
-bool monitor_stats::add_ap_metrics(ieee1905_1::CmduMessageTx &cmdu_tx, const sMacAddr &bssid)
+bool monitor_stats::add_ap_metrics(ieee1905_1::CmduMessageTx &cmdu_tx, const sMacAddr &bssid,
+                                   const size_t &sta_count, const uint8_t &channel_utilization)
```

Comment:
These should not be passed by const-ref but by value (const-ref only for objects that are larger than word size).

However, instead of passing in sta_count and channel_utilization, I think it's better to pass in vap_node (for sta_count) and radio_node (for channel utlization).

Reply:
Fixed